### DISCUSSION TITLE
麻烦精简功能

### DISCUSSION THREAD

**Discussion by simplebam** [community/community#28449] [Other Feature Feedback, Questions, & Ideas]
现在clash更新之后，功能越来越多，但其实我们只是想，输出某个服务器的IP链接，就可以访问外网了。

那么为什么要搞那么多选项出来呢？

**Comment by better-pz** [comment]
对呀,一直往上堆

**Comment by billythekid** [comment]
Hi @simplebam! 👋 

Firstly, I used a translate app to see what you were asking to see if I could help so apologies if I'm wrong on this, but I'm not sure you've raised this question in the correct place?

It looks like you're asking about a specific project that's hosted on GitHub rather than GitHub itself, and more specifically, your GitHub profile.

There are a few places we suggest you look when you need help with a project hosted on GitHub:

* its SUPPORT or README file, which may include contact information or an official website
* its Issues tab, where you can report bugs
* the project's mailing list (if it has one)
* the owner's GitHub profile, which may include contact information

I'd hate for you to not find the help you're looking for here so thought I should point that out for you. If you're not sure where the project lives if you can add some information here we might be able to help you find it.

Thanks,
Billy 🤠
